rust/ql/src/queries/security/CWE-825/AccessInvalidPointer.qhelp

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE qhelp PUBLIC
+  "-//Semmle//qhelp//EN"
+  "qhelp.dtd">
+<qhelp>
+<overview>
+
+<p>
+Dereferencing an invalid or dangling pointer is undefined behavior. Memory may be corrupted
```

Comment:
Perhaps I am being pedantic here, but I think we need to clarify 😅 
Dereferencing a pointer is not undefined behavior but it can cause undefined behavior.
 
```suggestion
Dereferencing an invalid or dangling pointer may cause undefined behavior. Memory may be corrupted
```